rails-observers.gemspec

```diff
@@ -19,9 +19,10 @@ Gem::Specification.new do |s|
   s.add_dependency 'activemodel', '>= 4.0'
 
   s.add_development_dependency 'minitest',     '>= 3'
-  s.add_development_dependency 'railties', '>= 4.0'
+  s.add_development_dependency 'railties',     '>= 4.0'
   s.add_development_dependency 'activerecord', '>= 4.0'
   s.add_development_dependency 'actionmailer', '>= 4.0'
-  s.add_development_dependency 'actionpack', '>= 4.0'
+  s.add_development_dependency 'actionpack',   '>= 4.0'
   s.add_development_dependency 'sqlite3',      '~> 1.3'
+  s.add_development_dependency 'activerecord-deprecated_finders'
```

Comment:
Yes. And if any test fail because of that we should remove all the usage of the deprecated finders.